locales/en.json

```diff
@@ -1,99 +1,93 @@
-    "stronglyDisagree": "Strongly Disagree",
-    "back": "Back",
-
-
-    "resultsHeaderOne": "The results",
-    "resultsHeaderTwo": "are",
-    "resultsHeaderThree": "in!",
-
-    "viewResults": "View Results",
-
-    "logo": "Logo",
-
-    "Anthropocentrism": "Anthropocentrism",
-    "Assimilationism": "Assimilationism",
-    "Authoritarianism": "Authoritarianism",
-    "Autocracy": "Autocracy",
-    "Accumulation": "Accumulation",
-    "Distribution": "Distribution",
-    "Conservatism": "Conservatism",
-    "Democracy": "Democracy",
-    "Ecocentrism": "Ecocentrism",
-    "Elitism": "Elitism",
-    "Globalism": "Globalism",
-    "Internationalism": "Internationalism",
-    "Isolationism": "Isolationism",
-    "Libertarianism": "Libertarianism",
-    "Meritocracy": "Meritocracy",
-    "Militarism": "Militarism",
-    "Modernism": "Modernism",
-    "Nationalism": "Nationalism",
-    "Pacifism": "Pacifism",
-    "Partisanship": "Partisanship",
-    "Pluralism": "Pluralism",
-    "Populism": "Populism",
-    "Secularism": "Secularism",
-    "Theocracy": "Theocracy",
-    
-
-    "placeholder": ""
+  "menu-home": "Home",
+  "menu-future-plans": "Future Plans",
+  "test-select-title": "Pick your test",
+  "test-select-complete-button": "Complete Test",
+  "test-select-complete-info": "(120 questions - 24 ideologies)",
+  "test-select-divider": "or",
+  "test-select-basic-button": "Basic Test",
+  "test-select-basic-info": "(40 questions - 8 ideologies)",
+  "footer-home": "Home",
+  "footer-home-features": "Features",
+  "footer-home-information": "Information",
+  "footer-home-goals": "Goals",
+  "footer-plans": "Future Plans",
+  "footer-plans-questions": "Questions",
+  "footer-plans-data": "Data",
+  "footer-plans-results": "Results",
+  "footer-test": "Take the Test",
+  "footer-test-complete": "Complete Test",
+  "footer-test-basic": "Basic Test",
+  "footer-feedback-title": "Give us your feedback",
+  "footer-feedback-submit": "Submit Feedback",
+  "banner-title": "Get a sense of your <br><span class=\"semi-bold\">ideology</span> with <span class=\"semi-bold\">Politivector</span>.",
+  "banner-subtitle": "A global test designed for a global audience to better comprehend where they stand.",
+  "banner-button-primary": "Get Started",
+  "banner-button-secondary": "Learn More",
+  "features-title": "All ideologies.<br><span class=\"semi-bold\">One political test.</span>",
+  "features-comprehensive-title": "Fully<br>comprehensive",
+  "features-comprehensive-text": "We evaluate your answers across 24 ideologies, the most numerous among common tests.",
+  "features-international-title": "International<br>questions",
+  "features-international-text": "Our questions are designed with global users in mind, no single country, no single culture.",
+  "features-anonymous-title": "Stay<br>anonymous",
+  "features-anonymous-text": "Information you explicitly provide is all we are going to get. No permission means no data.",
```

Comment:
( ͡° ͜ʖ ͡°)

Reply:
`  "plans-data-text": "Right now, Politivector obtains only your finalized test results, and the country from which you solved the test. Your country is obtained through your already publicly available information, and no further information is gathered. Your IP and device information are never stored. By gathering test results alongside country information, we wish to be able to compare test takers’ results —anonymous results, of course— and present a comprehensive image regarding the distribution of ideologies across the world. The information portrayed by such distribution and concentration maps could be especially useful in demographic, social, and political studies. <br><br>Another possibility is to apply various data science methods to the data provided to us by our test takers. One specific method is “clustering”. Clustering would allow us to group together data points, seeing if they agree on a common center. <br><br>These analyses could present many new perspectives regarding the status of political philosophy in the modern world, such as how fragmented certain ideologies are, which political parties best satiate their peoples’ needs, and where a newcomer into politics could align themselves as a starting point. <br><br>Data collection and analysis is probably the most intricate of our long-term goals, which is why it is the enhancement we wish to implement with utmost care.",
`

Reply:
"... Your country is determined through the IP address your browser already provides us when you connect to connect to a website, but the IP address itself or any device information are never stored ..."

Reply:
Your country is determined through the IP address your browser already provides us when you connect to our website, but the IP address itself or any device information are never stored.